packages/teams-js/src/private/hostEntity/tab.ts

```diff
@@ -116,6 +123,45 @@ class HostEntityTabInstancesResponseHandler extends ResponseHandler<HostEntityTa
  * runs through app configuration if required, and then associates the app with the threadId provided
  *
  * @param hostEntityIds Ids of the host entity like channel, chat or meeting
+ * @param appTypes -  What type of applications to show the user. If EDU is passed as appType, only apps supported by EDU tenant are shown.
+ * If no value is passed, all apps are shown.
+ * @returns The HostEntityTabInstance of the newly associated app
+ */
+export function addAndConfigure(hostEntityIds: HostEntityIds, appTypes?: AppTypes[]): Promise<HostEntityTabInstance>;
+
+/**
+ * @hidden
+ * @internal
+ * @beta
+ * Limited to Microsoft-internal use
+ *
+ * Launches host-owned UI that lets a user select an app, installs it if required,
+ * runs through app configuration if required, and then associates the app with the threadId provided
```

Comment:
Could you please add another sentence here to explain what makes this overloaded function different than the original?